src/Raven.Studio/typings/_studio/dto.d.ts

```diff
@@ -317,12 +317,33 @@ interface layoutable {
 
 
 interface autoCompleteWordList {
-    caption: string; 
+    caption?: string; 
     value: string; 
     score: number; 
     meta: string 
 }
 
+interface autoCompleteLastKeyword {
+    keywordsBefore: string[],
```

Comment:
this struct becomes too complex imho...

Reply:
I would be probably simplified later on.